### ISSUE TITLE
Reset methods

### ISSUE THREAD

**Issue by ctrachte** [ctrachte/Kitchen-Calendar#3]
Need to be able to reset calendar to current month

**Comment by Pinaka-Pani-18** [comment]
Can I work on this issue

**Comment by ctrachte** [comment]
Yes, but I need to merge one more PR first. As soon as I have done that, I will assign to you. Should be tomorrow morning at the earliest.

**Comment by ctrachte** [comment]
assigned, be sure to make changes on a new branch, and submit them via PR, have fun!
